src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -713,6 +713,7 @@ def load_arguments(self, _):
         c.argument('reset', help='Reset Java apps to the default parking page if set to true with no type specified.', arg_type=get_three_state_flag())
         c.argument('timeout', type=int, help='Timeout for the deployment operation in milliseconds. Ignored when using "--src-url" since synchronous deployments are not yet supported when using "--src-url"')
         c.argument('slot', help="The name of the slot. Default to the productions slot if not specified.")
+        c.argument('track_status', help="If true, web app runtime status during deployment will be tracked.", arg_type=get_three_state_flag())
```

Comment:
In the help, should we highlight, `deployment will be tracked for linux webapps.